apps/api/src/app/shared/helpers/e2e/sdk/e2e-sdk.helper.ts

```diff
@@ -1,17 +1,17 @@
 import { Novu } from '@novu/api';
 import { NovuCore } from '@novu/api/core';
-import { UserSession } from '@novu/testing';
-import { expect } from 'chai';
-import { ErrorDto, SDKValidationError, ValidationErrorDto } from '@novu/api/models/errors';
 import { SDKOptions } from '@novu/api/lib/config';
 import { HTTPClient, HTTPClientOptions } from '@novu/api/lib/http';
+import { ErrorDto, SDKValidationError, ValidationErrorDto } from '@novu/api/models/errors';
 import { HttpRequestHeaderKeysEnum } from '@novu/application-generic';
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
 
 export function initNovuClassSdk(session: UserSession, shouldRetry: boolean = false): Novu {
   const options: SDKOptions = {
     security: { secretKey: session.apiKey },
     serverURL: session.serverUrl,
-    debugLogger: console,
+    debugLogger: process.env.LOG_LEVEL === 'debug' ? console : undefined,
```

Comment:
👏👏
what does the sdk logs 😅

Reply:
A lot of noise, every request and response pretty much